contrib/start_development_worker

```diff
@@ -8,7 +8,11 @@ function _print_syntax() {
   me=`basename "$0"`
   echo "Usage: $me [-d <dir> -w <num>]"
   echo -e "\t-d <dir>\tDirectory of source code. (default: /obs)"
+  echo -e "\t-l <dir>\tDirectory where the logfiles should be put. If not set STDERR and STDOUT will be printed to console"
```

Comment:
💭 just out of curiosity, why do you need to redirect the logs?

Reply:
If i have larger development setups I want to be able to look at seperate log files instead of scrolling through the console.